core/local/local.go

```diff
@@ -115,6 +124,11 @@ func (e *ExecutionScheduler) GetExecutors() []lib.Executor {
 	return e.executors
 }
 
+// GetExecutorConfigs returns the slice of executor configs.
```

Comment:
```suggestion
// GetExecutorConfigs returns the slice of all executor configs, sorted by
// their (startTime, name) in an ascending order.
```

You may also want to adjust the comment of `GetExecutors()` to mention that it returns only the executors which have some work.

Reply:
Fixed